docs/ml-features.md

```diff
@@ -211,6 +211,65 @@ for more details on the API.
 </div>
 </div>
 
+## FeatureHasher
```

Comment:
I put the doc under the Feature Extractors section, let me know if it's not the right place